### ISSUE TITLE
Allow null overrideAlias in query functions

### ISSUE THREAD

**Issue by Swahvay** [lolopinto/ent#1838]
There is currently no way to _remove_ an alias with `aliasOverride` in [simpleClause](https://github.com/lolopinto/ent/blob/4b6b14425c8c1add6bfc3a3e0ab674ace5514659/ts/src/core/clause.ts#L59). This is useful in cases where I do some augmenting on the field name like `query.eq('LOWER(name)', $name)`. If I am using this clause in a `CustomClauseQuery` that joins with another table and has specified an `alias`, there is no way to remove that alias for this clause. It ends up compiling to `alias.LOWER(name) = $1`. I can get around this by specifying `fieldAlias` in `loadEntOptions` and then manually setting the `tableName` to `'table_name alias'`. Kinda hacky but technically works fine.

I'm suggesting changing the `overrideAlias` definition from `overrideAlias?: string` to `overrideAlias?: string | null`. The other solution I thought of was something like a new parameter that would wrap column names. So it would be something like `query.Eq('name', $name, { columnOp: 'LOWER' })`.

**Comment by lolopinto** [comment]
makes sense. just to jog my brain, how does `LOWER(name)` know which table to use?

**Comment by Swahvay** [comment]
In this example `name` is just the column name, so if you were using an alias you would have to do `LOWER(alias.name)`.
